### ISSUE TITLE
No way to login after choosing local storage

### ISSUE THREAD

**Issue by adom44** [alainm23/planify#878]
I downloaded planner on one pc first and chose a local, for I wanted to first try out the app before making an account.
I later downloaded it on a second pc and logged in. After that I found out that the app has no way to login after that one choice on first opening of the app. There is simply no way to sync my two pcs planner apps now. The choice between login and local shows up for barely a milisecond before checking what choice was initially done and instantly closing. Reinstalling the app doesnt fix the issue. It probably keeps that choice somewhere else. Cant find the app anywhere on the disk to delete its file. Who knows, what name is it hiding under there.

TLDR: Need a way to login in to my account in the app even after I already chose local storage on first opening. Just adding a button "login" in the settings would be plenty enough.

**Comment by adom44** [comment]
After about a million times, I did actually manage to click on login before the choices closed, so my issue is sorta closed. I will keep this issue opened here though, for the login could for sure be easier to get to.

This is and update proposal, after choosing local, have an option in the app to sync with other pc.

**Comment by elvindsouza** [comment]
Can confirm this occurs on 

uname -a
`>Linux PC 5.13.0-30-generic #33-Ubuntu SMP Fri Feb 4 17:03:31 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`

Planner flatpak installed
`Flatpak 1.10.2`
`Planner 3.0.6`

lsb_release --all
```Distributor ID: Ubuntu
Description:    Ubuntu 21.10
Release:        21.10
Codename:       impish
```

Happened after updating to new Planner major version
Solved by purging all data(`flatpak uninstall --delete-data`), and was able to restore old tasks as I use the Todoist integration(phew)

**Comment by joeljose1001** [comment]
Facing the same issue. 
Removing the .db file from .local/share/com.github.alainm23.planner and restarting the app makes the options show up momentarily but that's all I could manage. Unable to select the options. Reinstalling the application did not help.
Hoping for a fix in later releases. Thanks.

**Comment by soliveirarm** [comment]
I had the same problem, chose the local option but I've changed my mind and wanted to sync with Todoist, now I can't cuz the option won't show up

**Comment by alainm23** [comment]
Try:
`flatpak run com.github.alainm23.planner -r`

**Comment by lilithium-hydride** [comment]
Can this be reopened? @joeljose1001 reported that resetting the application does not fix the issue, and this has been my experience as well. 

The output of `strace -e trace=openat planner` shows that it's looking at dconf for something, and browsing to `/com/github/alainm23/planner3/` in dconf Editor does indeed reveal that it's leaving stuff there. Setting `backend-type` to `'none'` and relaunching Planner allows you to select a different backend as expected. This can also be done with `dconf reset /com/github/alainm23/planner3/backend-type'` if you don't want to install the GUI editor.

Resetting the application should probably wipe the dconf stuff as well. It would be a lot more intuitive to store this sort of stuff in `$XDG_CONFIG_HOME` as a normal file, though. Easily synced and backed up, very obviously present when searching for leftover application data, programmatically modifiable/removable by standard utilities, etc.

**Comment by alainm23** [comment]
If you want to start again, you can enter the preferences and select the option 'Delete all my app data'
![image](https://user-images.githubusercontent.com/33765137/167661666-c63b3f67-7fe4-425e-9e00-f2d50742976d.png)

**Comment by lilithium-hydride** [comment]
That appears to do the same thing as `planner -r`, which does not reset or delete the appropriate dconf entries.
